igor-web/src/main/groovy/com/netflix/spinnaker/igor/build/BuildController.groovy

```diff
@@ -184,10 +184,7 @@ class BuildController {
 
             queuedLocation.split('/')[-1]
         } else if (buildMasters.filteredMap(BuildServiceProvider.TRAVIS).containsKey(master)) {
-            if (requestParams) {
-                return buildMasters.map[master].triggerBuildWithParameters(job, requestParams)
-            }
-            return buildMasters.map[master].triggerBuild(job)
+            return buildMasters.map[master].triggerBuildWithParameters(job, requestParams)
```

Comment:
yes
```
curl -X PUT "localhost:8080/masters/travis-ci/jobs/gardalize/travis-trigger-test/master?foo=bar&bar=foo"
1495209235
```
https://travis-ci.org/gardalize/travis-trigger-test/jobs/234070742/config

```
curl -X PUT "localhost:8080/masters/travis-ci/jobs/gardalize/travis-trigger-test/master"
1495209330
```
https://travis-ci.org/gardalize/travis-trigger-test/jobs/234071363/config


So it handles triggering with parameters and without.